File: Atividade 5 V1/Quest7.pas
Program mediaarit_idadedapop ;
    var idade,soma,qtdade:integer;
        media:real;
        Begin
            soma:=0;
            qtdade:=0;
            write('Informe a idade do individuo');
            read(idade);
            while(idade<>0) do
                begin
                    soma:=soma+idade;
                    qtdade:=qtdade+1;
                    write('Informe a idade do individuo');
                    read(idade);
                end;
		  media:=(soma/qtdade);
		  write((media):2:2);   
	   End.

File: Alg2-A1/Quest7.c
#include <stdio.h>

void main(){
	int idade,soma;
	float qtdade,media;
	soma=0;
	qtdade=0;
	printf("Informe o valor da idade do individuo= ");
	scanf("%d",&idade);
	while(idade!=0){
		qtdade=qtdade+1;
		soma=soma+idade;	
		printf("Informe a idade do individuo= ");
		scanf("%d",&idade);	
	}
	media=soma/qtdade;
	printf("%.2f",media);
}
